ocaml/quicktest/quicktest_cbt.ml

```diff
@@ -123,7 +123,7 @@ let vdi_data_destroy_test ~session_id ~vDI =
       ~msg:"VDI.data_destroy failed to update VDI.type";
     assert_cbt_status true ~session_id ~test ~vDI:snapshot
       ~msg:"VDI snapshot cbt_enabled field erroneously set to false";
-  (*  test_vdi_update ~session_id ~test snapshot; 
+  (*  test_vdi_update ~session_id ~test snapshot;
```

Comment:
Did you intentionally reindent code here? If not might be better to send a "reindent XAPI" pull request to master fist by running `make reindent`, rather than reindenting files in each PR. Then we can rebase from master with `-Xignore-space-change` and continue our work without having to reindent things.
The checklist from the ticket contains a reindent item as well.

Reply:
Not intentionally, I forgot to turn off my vim plugin which automatically strips whitespace.

Reply:
Reindentation PR (here[https://github.com/xapi-project/xen-api/pull/3506])